### PR TITLE
Simple upload of temp basal records to NS

### PR DESCRIPTION
**Pull request by ps2**

### CODE REVIEW

File: MinimedKit/PumpEvents/TempBasalPumpEvent.swift [hunk 1]
```diff
@@ -9,10 +9,17 @@
 import Foundation
 
 public class TempBasalPumpEvent: PumpEvent {
+  
+  public enum RateType : String {
+    case Absolute = "absolute"
```
Comment (loudnate) [inline MinimedKit/PumpEvents/TempBasalPumpEvent.swift:14]: Are these raw enumerations part of the Nightscout model? If not, and they're just for debugging, Swift will automatically stringify enum cases when you extend `String`:

``` swift
enum RateType: String {
    case Absolute  // = "Absolute"
    case Percent  // = "Percent"
}
```

**Comment by loudnate** [comment]
LGTM

File: RileyLink/NightscoutUploadKit/NightscoutUploader.swift [hunk 2]
```diff
@@ -526,14 +471,58 @@ class NightScoutUploader: NSObject {
...
+          }
+        })
+        task.resume()
       } catch {
-        NSLog("Couldn't encode data to json.");
+        completion("Couldn't encode data to json.")
       }
     } else {
-      NSLog("Invalid URL: %@, %@", siteURL, endpoint)
+      completion(String(format:"Invalid URL: %@, %@", siteURL, endpoint))
```
Comment (loudnate) [inline RileyLink/NightscoutUploadKit/NightscoutUploader.swift]: Use interpolation here: 

``` swift
completion("Invalid URL: \(siteURL), \(endpoint)")
```

File: MinimedKit/PumpEvents/TempBasalPumpEvent.swift (hunk 1 above)
Comment (ps2) [inline MinimedKit/PumpEvents/TempBasalPumpEvent.swift:14]: Those are part of the ns model; I wanted lowercase.

**Pull request merged**